File: 3190-FindMinimumOperationsToMakeAllElementsDivisibleByThree/3190-FindMinimumOperationsToMakeAllElementsDivisibleByThree.py
# Last updated: 9/14/2026, 11:54:58 PM
class Solution(object):
    def minimumOperations(self, nums):
        y=0
        see=len(nums)

        for i in range(see):
            if nums[i]%3==0:
                  y+=1
        minus=y-see
        if minus<0:
             return (minus*-1)
        else:
             return (minus)

File: 3190-FindMinimumOperationsToMakeAllElementsDivisibleByThree/3190-FindMinimumOperationsToMakeAllElementsDivisibleByThree.c
// Last updated: 9/14/2026, 11:54:52 PM
int minimumOperations(int* num, int n) {
    int c=0;
    for(int i=0;i<n;i++)
    {
        if(num[i]%3!=0)
        c=c+1;
    }
    return c;
}